components/inputmethod/libhangul/Makefile

```diff
@@ -0,0 +1,46 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2025 Jiwon Na
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=           libhangul
+COMPONENT_VERSION=        0.1.0
+COMPONENT_GIT_REV=	154a5e0
```

Comment:
The exisiting version of libhangul is 0.5.11 which is > 0.1.0.
So you need to add
IPS_COMPONENT_VERSION=	0.5.11
and should also add
COMPONENT_REVISION= 1
(which is probably not strictly needed)